docs/lib/datastore/fragments/ios/auth-model.md

```diff
@@ -0,0 +1,104 @@
+<amplify-callout>
+When using the `@auth` directive with DataStore, currently only following fields are supported:
+
+1. `allow` with "owner" and "groups"
+2. `ownerField`
+3. `groups`
+4. `operations`
+
+</amplify-callout>
+
+## Updated schema
+
+Create a Model that is read-only.
+
+```graphql
+type Todo
+    @model
+    @auth(rules: [
+        { allow: owner, ownerField: "owner", operations: [create, update, delete] },
+    ]) {
+    id: ID!
+    content: String!
+    owner: String
+}
+```
+
+The API will allow users to query, list, and subscribe to model creation, updates, deletions by other users. A user can only update and delete their own model, that is a model can not be updated or deleted by another user.
+
+## API with Cognito User Pool auth
+
+With the schema, make sure you have selected Cognito User Pool as the auth
```

Comment:
We need a lead-in sentence here to help users flow from "schema" thinking to "auth" thinking.  How about something like.
While using the @auth directive in your schema to limit read/write access, we require authentication to be integrated into your app in order to differentiate who the owner is.  To set this up issue the command:
```
amplify add auth
```
then follow the prompts:

```
prompts here
```